src/main/resources/static/number.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+</head>
+<body>
+<p>Here should show a number with proper URL code</p>
+<img id="image" src="/api/number/">
```

Comment:
maybe just remove src here, since it's replaced by script? (it doesn't point to any real picture, so it might be misleading to keep it.

Reply:
ok